packages/components/tests/e2e/specs/appmap/sequenceDiagram.spec.js

```diff
@@ -46,15 +46,30 @@ context('AppMap sequence diagram', () => {
       cy.get('.sequence-actor[data-actor-id="class:openssl/OpenSSL::Cipher"]').should('exist');
     });
 
-
-    it.only('should display the tooltip on hover', () => {
+    it('should display the tooltip on hover on return label', () => {
       // Hover over the .label
       cy.get('.return:nth-child(14) .name').trigger('mouseover');
       cy.get('.return:nth-child(14) span.tooltip').should('exist');
       cy.get('.return:nth-child(14) .name').trigger('mouseout');
       cy.get('.return:nth-child(14) span.tooltip').should('not.exist');
     });
 
+    it('should display the tooltip on hover on call label', () => {
+      // Hover over the .label
+      cy.get('.call:nth-child(17) .name').trigger('mouseover');
+      cy.get('.call:nth-child(17) span.tooltip').should('exist');
+      cy.get('.call:nth-child(17) .name').trigger('mouseout');
+      cy.get('.call:nth-child(17) span.tooltip').should('not.exist');
+    });
+
+    it('should display the tooltip on hover on link_to_with_icon call label', () => {
+      // Hover over the .label
+      cy.get('.call:nth-child(72) .name').trigger('mouseover');
+      cy.get('.call:nth-child(72) span.tooltip').should('exist');
+      cy.get('.call:nth-child(72) .name').trigger('mouseout');
+      cy.get('.call:nth-child(72) span.tooltip').should('not.exist');
+    });
+
```

Comment:
Can you add tests that check the text content of the hover to ensure that the `truncate` function is working correctly?

Also, can you add tests that check that there is no hover state for events that are missing `currentState` or events that don't have parameters.

Reply:
Added testcases.